src/content/docs/data-apis/manage-data/drop-data-using-nerdgraph.mdx

```diff
@@ -51,19 +52,17 @@ The following data types can be targeted for data dropping:
 * Custom events (like those generated by the [APM agent APIs](/docs/insights/insights-data-sources/custom-data/insert-custom-events-new-relic-apm-agents) or the [Event API](/docs/insights/insights-data-sources/custom-data/introduction-event-api))
 * Log data (you can also [use the UI to drop data](/docs/logs/ui-data/drop-data-drop-filter-rules))
 * Distributed tracing spans
-* [Default infrastructure monitoring events](/docs/infrastructure/manage-your-data/data-instrumentation/default-infrastructure-monitoring-data) and [infrastructure integrations](/docs/integrations/infrastructure-integrations/get-started/introduction-infrastructure-integrations) events, with these caveats:
-  * Aggregated, non-raw `SystemSample`, `ProcessSample`, `NetworkSample` and `StorageSample` can't be dropped (for more on this, see [Data retention](/docs/data-apis/manage-data/manage-data-retention/#infrastructure-data)). This data doesn't count towards your data ingest.
-  * Raw infrastructure data is used for alerting, so if you are dropping these events, you will not be able to alert on them even if you see data in charts with time windows longer than 59 minutes (such as alert condition preview charts).
-* [Dimensional metrics](/docs/telemetry-data-platform/ingest-manage-data/understand-data/new-relic-data-types#dimensional-metrics), with these caveats:
-  * Billing impacts: for [our usage-based pricing](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing), dropped data is not billable. For [our original pricing model](/docs/accounts/original-accounts-billing/product-pricing/product-based-pricing), dropped data is billable.
+* [Default infrastructure monitoring events](/docs/infrastructure/manage-your-data/data-instrumentation/default-infrastructure-monitoring-data) and [infrastructure integrations](/docs/integrations/infrastructure-integrations/get-started/introduction-infrastructure-integrations) events. Some caveats:
+  * When you drop this data, the raw data is dropped, but the aggregated `SystemSample`, `ProcessSample`, `NetworkSample` and `StorageSample` is still available (for more on this, see [Data retention](/docs/data-apis/manage-data/manage-data-retention/#infrastructure-data)). Though still available, this data doesn't count towards ingest and is not billable.
```

Comment:
@zuluecho9 subject-verb agreement: "...the aggregated...ARE still available...."